crates/libs/bindgen/src/gen.rs

```diff
@@ -277,7 +277,7 @@ impl<'a> Gen<'a> {
     }
     /// The signature params which are generic (along with their relative index)
     pub fn generic_params<'b>(&'b self, params: &'b [SignatureParam]) -> impl Iterator<Item = (usize, &SignatureParam)> + 'b {
```

Comment:
I guess this method should also change names to `convertible_params` no? Or can this be kept as `generic_params`  if we add the check for `signature_param_input_value` here?